tools/generator/SourceWriters/InterfaceInvokerProperty.cs

```diff
@@ -27,6 +27,8 @@ public InterfaceInvokerProperty (InterfaceGen iface, Property property, CodeGene
 			IsPublic = true;
 			IsUnsafe = true;
 
+			SourceWriterExtensions.AddSupportedOSPlatform (Attributes, property.Getter, opt);
```

Comment:
Do we have any instances where property getter & setter are in two separate API levels?  Is this something we need to worry about or consider?

Reply:
I'm not aware of any, and at least for interfaces we probably wouldn't allow it to happen, as adding a required setter in a later API would be a breaking change.  I think we can ignore this.